crates/context_aware_config/src/db/models.rs

```diff
@@ -74,3 +76,15 @@ pub struct EventLog {
     pub new_data: Option<Value>,
     pub query: String,
 }
+
+#[derive(Queryable, Selectable, Insertable, AsChangeset, Clone, Serialize, Debug)]
+#[diesel(check_for_backend(diesel::pg::Pg))]
+#[diesel(table_name = jsonschema_types)]
+#[diesel(primary_key(id))]
```

Comment:
```suggestion
#[diesel(primary_key(type_name))]
```

there is no `id` column in `jsonschema_types` table

Reply:
Nice, I missed this